src/components/Button.component.tsx

```diff
@@ -0,0 +1,82 @@
+import React,{FC} from 'react'
+import styled from 'styled-components';
+
+import {ButtonProps} from './Button.types'
+import {ButtonColor, ButtonSizes} from "./Button.constants";
+
+const StyledButton = styled.button<ButtonProps>`
+  
+    position: relative;
+	padding: .7em 1.4em;
+	width: fit-content;
+	font-size: ${props => props.size == ButtonSizes.SMALL ? '10px' : (props.size == ButtonSizes.MEDIUM ? '15px' : '20px' )  };
+	color: ${props => props.primary ? props.secondaryColor : props.primaryColor};
```

Comment:
It should be color: ${props => props.primary ? props.primaryColor : props.secondaryColor };